.java-version

```diff
@@ -1 +1 @@
-17
+17.0
```

Comment:
I needed to do this to get ECM to run on my machine! I will remove it if this isn't generally applicable.